WordPress/Classes/ViewRelated/Media/AnimatedImageCache.swift

```diff
@@ -23,50 +19,38 @@ class AnimatedImageCache {
         return session
     }()
 
-    fileprivate static var cache: NSCache<AnyObject, AnyObject> = {
-        let cache = NSCache<AnyObject, AnyObject>()
-        cache.totalCostLimit = totalCostLimit()
-
-        let bcf = ByteCountFormatter()
-        bcf.allowedUnits = [.useMB]
-        bcf.countStyle = .file
-        let cacheSizeinMB = bcf.string(fromByteCount: Int64(totalCostLimit()))
-
-        return cache
-    }()
-
     // MARK: Instance methods
 
-    @objc func handleMemoryWarning() {
-        clearCache()
-    }
-
-    func clearCache() {
-        AnimatedImageCache.cache.removeAllObjects()
+    func cacheData(data: Data, url: URL?) {
+        guard let url = url else {
```

Comment:
nitpick: Does shortening to this make sense?

```
guard let url else {
```

Reply:
I think the Xcode was just recently updated on the CI to the version that supports this new shorthand, so I continued using the longer form. Updated. The shorter version is much nicer.